win32ss/user/ntuser/window.c

```diff
@@ -767,7 +768,7 @@ LRESULT co_UserFreeWindow(PWND Window,
 //   ASSERT(Window != NULL);
```

Comment:
While here, this commented out `ASSERT()` can be removed, or at least moved to a more useful place.
It is as is since 11291ff (r49109).

Reply:
> While here

I'm not making a trip. This code has nothing to do with this PR.